Gradle 配置引入 properties 的方式对其他获取源码的开发者不友好

https://github.com/jovezhao/automapper/blob/8707f15c296d524a3cc8e151a3975508c922e67e/publish.gradle#L59-L71

缺少 properties 配置的环境会出现：
```
Starting Gradle Daemon...
Gradle Daemon started in 557 ms

FAILURE: Build failed with an exception.

* Where:
Script '/home/sia/IdeaProjects/automapper/publish.gradle' line: 66

* What went wrong:
A problem occurred evaluating script.
> Could not get unknown property 'NEXUS_USERNAME' for Credentials [username: null] of type org.gradle.internal.credentials.DefaultPasswordCredentials_Decorated.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.1.1/userguide/command_line_interface.html#sec:command_line_warnings
```

--- a/publish.gradle
+++ b/publish.gradle
@@ -63,8 +63,8 @@ publishing {
             // 如果version以SNAPSHOT结尾就会上传到快照仓库，如果不是就上传到release仓库
             url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
             credentials {
-                username = "${NEXUS_USERNAME}"
-                password = "${NEXUS_PASSWORD}"
+                username = project.properties.NEXUS_USERNAME
+                password = project.properties.NEXUS_PASSWORD
             }
         }
 

